packages/table/components/Column.tsx

```diff
@@ -14,7 +18,11 @@ export interface ColumnProps {
   /**
    * cellRenderer is the function which is creating the cell contents for this column.
    */
-  cellRenderer: (data: any, width: number) => React.ReactNode;
+  cellRenderer: (
+    data: any,
+    width: number,
+    additionalProps: OverloadedProps
```

Comment:
Is there a reason this can't be optional?

Reply:
It should at least be an empty object

Reply:
This will break existing tables in dcos-ui, then?

Reply:
It doesn't, I might have misunderstood.

Reply:
It shouldn't, you can just ommit the argument in the cell renderer :)